### ISSUE TITLE
A user should not be able to use passwords with all possible characters

### ISSUE THREAD

**Issue by BlueRyth** [partier/emily#34]

**Comment by zachwlewis** [comment]
¿ʇuɐʍ ʎǝɥʇ ɹǝʇɔɐɹɐɥɔ ʎuɐ buıɹǝʇuǝ ɯoɹɟ sɹǝsn buıʇuǝʌǝɹd ɟo ǝbɐʇuɐʌpɐ ǝɥʇ s,ʇɐɥʍ

**Comment by BlueRyth** [comment]
I mean this more in the context of sanitization, we can technically let them use anything they want, but we're not storing in a format that accepts literally everything. UTF-8 is pretty all encompassing, but still.

**Comment by zachwlewis** [comment]
Okay. Just making sure we weren't pulling some, "No numbers or special characters and 8 letters max" bullshit.

**Comment by BlueRyth** [comment]
No, I'll hold whatever limit we decide. 256? Whatever symbols that don't interfere with santization, no leading quotes cause fuck that, and whatever else.
